AraEvent/AraEventCalibrator.cxx

```diff
@@ -13,6 +13,9 @@
 #include "araSoft.h"
 #include "TMath.h"
 #include "TGraph.h"
+#include "TMatrixD.h"
+#include "TVectorD.h"
+#include "TDecompSVD.h"
```

Comment:
note that the ROOT built-in SVD decomposition is very slow in my experience compared to others (e.g. Eigen3).

Reply:
It might be worth having this be settable at compile-time if it takes significant time.

Reply:
Thanks, didn't know that about the ROOT SVD decomposition. I'll try implementing this in Eigen3 to see if it's better. And agreed we should have an easy way to bypass this if it's too slow. 

Maybe just adding another calibration option is a little easier than a compile-time option?